db/migrate/20200319233723_add_deleted_at_to_identities.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeletedAtToIdentities < ActiveRecord::Migration[5.1]
+  def change
+    add_column :identities, :deleted_at, :datetime, null: true
```

Comment:
No need for an index here? I don't think there is but double checking.

Reply:
No need

Reply:
I think there is no need, we access this by user, and so I think the number of things that are deleted (and the number of time we query to look for deleted) are pretty minimal